src/main/resources/processes/IEDET/IEDET_PSU_TRIAGE.bpmn

```diff
@@ -1,25 +1,105 @@
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_1t13r39" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.9.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.15.0">
   <bpmn:process id="IEDET_PSU_TRIAGE" isExecutable="true">
     <bpmn:startEvent id="StartEvent_Triage">
       <bpmn:outgoing>Flow_18d0rzg</bpmn:outgoing>
     </bpmn:startEvent>
     <bpmn:endEvent id="EndEvent_Triage">
-      <bpmn:incoming>Flow_18d0rzg</bpmn:incoming>
+      <bpmn:incoming>Flow_13xu4ud</bpmn:incoming>
+      <bpmn:incoming>Flow_09pdaeg</bpmn:incoming>
+      <bpmn:incoming>Flow_1gidqty</bpmn:incoming>
     </bpmn:endEvent>
-    <bpmn:sequenceFlow id="Flow_18d0rzg" sourceRef="StartEvent_Triage" targetRef="EndEvent_Triage" />
+    <bpmn:sequenceFlow id="Flow_18d0rzg" sourceRef="StartEvent_Triage" targetRef="Screen_PSUComplaints" />
+    <bpmn:userTask id="Screen_PSUComplaints" name="Complaint Type" camunda:formKey="PSU_TRAIGE_COMPLAINT">
+      <bpmn:incoming>Flow_18d0rzg</bpmn:incoming>
+      <bpmn:outgoing>Flow_12yecwj</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:sequenceFlow id="Flow_12yecwj" sourceRef="Screen_PSUComplaints" targetRef="Gateway_0khr7oc" />
+    <bpmn:exclusiveGateway id="Gateway_0khr7oc" default="Flow_13xu4ud">
+      <bpmn:incoming>Flow_12yecwj</bpmn:incoming>
+      <bpmn:outgoing>Flow_13xu4ud</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0rvtbmq</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1gidqty</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_13xu4ud" sourceRef="Gateway_0khr7oc" targetRef="EndEvent_Triage" />
+    <bpmn:sequenceFlow id="Flow_0rvtbmq" sourceRef="Gateway_0khr7oc" targetRef="Activity_0dgy2k0">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CompliantType == "ReturnCase"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:serviceTask id="Activity_0dgy2k0" name="Reset IE Complaint Categories" camunda:expression="${bpmnService.blankCaseValues(execution.getVariable(&#34;CaseUUID&#34;), execution.processBusinessKey, &#34;CompCategoriesService&#34;, &#34;CompCategoriesSeriousAndMinor&#34;, &#34;CompCategoriesSerious&#34;)}">
```

Comment:
These fields have been changed in HOCS-5726, so this will need updating to blank the new values

Reply:
Thanks for pointing it out and I have updated my code/tests accordingly.